Libplanet/Blockchain/BlockChain.cs

```diff
@@ -557,6 +557,7 @@ public long GetNextTxNonce(Address address)
                 .IterateStagedTransactionIds()
                 .Select(Store.GetTransaction<T>)
                 .Where(tx => tx.Nonce < GetNextTxNonce(tx.Signer));
+            Block<T> prevBlock = Store.GetBlock<T>(prevHash.Value);
```

Comment:
Since `prevHash` is `nullable`, we have to prevent error that may caused by the case. I suggest as follow:
```suggestion
            Block<T> prevBlock = prevHash is null ? null : Store.GetBlock<T>(prevHash.Value);
```